packages/main/src/themes/Bar.css

```diff
@@ -0,0 +1,52 @@
+:host {
+	background-color: var( --sapPageHeader_Background);
+	height: var(--_ui5_bar_base_height);
+	width: 100%;
+}
+
+.ui5-bar {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	height: inherit;
+	width: inherit;
+	background-color: inherit;
+	margin-bottom: 0.5rem;
+	margin-top: 0.5rem;
+}
+
+.ui5-bar .ui5-bar-leftcontent-container {
+	padding-left: 0.5rem;
+}
+
+.ui5-bar .ui5-bar-rightcontent-container {
+	padding-right: 0.5rem;
+}
+.ui5-bar .ui5-bar-midcontent-container {
+	padding-left: 0.5rem;
+	padding-right: 0.5rem;
+}
+
+.ui5-bar.ui5-bar-design-bottomborder{
```

Comment:
The designs can be implemented this way:

:host([design="BottomBorder"]) {}

or if you should style the root div

:host([design="BottomBorder"]) .ui5-bar {}